boards/arm/nrf5340dk_nrf5340/Kconfig.defconfig

```diff
@@ -67,11 +67,11 @@ config MBOX_NRFX_IPC
 if BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP_NS
 
 choice BT_HCI_BUS_TYPE
-	default BT_RPMSG if BT
+	default BT_HCI_IPC if BT
 endchoice
 
 config HEAP_MEM_POOL_SIZE
-	default 4096 if BT_RPMSG
+	default 4096 if BT_HCI_IPC
```

Comment:
I'm curious if the heap is required for BT_HCI or for one of the HCI backends

Reply:
Not sure, this was introduced in 2cceb6220642928e08657e63736fab6cd5e0a6b7. @kapi-no do you know if this is needed?